appstudio-utils/util-scripts/lib/fetch/data.sh

```diff
@@ -6,17 +6,33 @@
 #  $DATA_DIR/foo/bar/123/data.json
 #
 json-data-file() {
-  local dir="$DATA_DIR"
+  rego-input-file "data" "$DATA_DIR" "$@"
+}
+
+# Each argument will become a directory, e.g. if you call it like this:
+#  json-input-file foo bar 123
+#
+# The result will be this:
+#  $INPUT_DIR/foo/bar/123/input.json
+#
+json-input-file() {
+  rego-input-file "input" "$INPUT_DIR" "$@"
+
+}
```

Comment:
The purpose of this method was so you could easily nest directories, as many as you want. Since there's just one input file, I don't think there's any need to use this method now. We can just send data to $INPUT_FILE directly.